docs/indepth/installation.rst

```diff
@@ -225,7 +227,7 @@ Using Amazon Web Services
       :prompt: [root@aws]#
 
 
-#. Follow the :ref:`generic Fedora 20 installation instructions <fedora-20-install>` below.
+#. Follow the operating system specific installation instructions below.
```

Comment:
This implies that we support all the operating systems below on DO, but we only support Fedora 20 for now, right?